### DISCUSSION TITLE
Is it possible to specify the drag area of the item?

### DISCUSSION THREAD

**Discussion by jullyb** [jbaysolutions/vue-grid-layout#584] [Q&A]
Hi, i have a very complex item, with many controls and a header.
Now if my mouse over the whole item (not only over header) i see the drag cursor and the logic of all controls are broken.
The idea to specify the area where the drag action will be activated (like for resize - the bottom edge) - for example for the header.

Maybe it's already possible?

**Comment by gmsa** [comment]
Read the docs: https://jbaysolutions.github.io/vue-grid-layout/guide/properties.html#dragallowfrom
Example: https://jbaysolutions.github.io/vue-grid-layout/guide/04-allow-ignore.html

**Reply by jullyb** [reply]
thanks
